src/specs/package.lisp

```diff
@@ -98,81 +98,98 @@ constructors"
-           (rty geb.spec:substmorph)
-           (cod geb.spec:substmorph)
-           (on t) (left t) (right t))
-  (pair (lty geb.spec:substmorph) (rty geb.spec:substmorph) (left t) (right t))
-  (fst  (lty geb.spec:substmorph) (rty geb.spec:substmorph) (value t))
-  (snd  (lty geb.spec:substmorph) (rty geb.spec:substmorph) (value t))
-  (lamb (vty geb.spec:substmorph) (tty geb.spec:substmorph) (value t))
-  (app  (dom geb.spec:substmorph) (cod geb.spec:substmorph) (func t) (obj t))
-  (index (index fixnum)))
-```
+  typed lambda calculus within GEB. The class presents untyped STLC terms.
 "
-  (<stlc> pax:type)
-  (stlc pax:type)
-  (absurd pax:type)
-  (absurd-value pax:function)
-
-  (unit pax:type)
-
-  (pair pax:type)
-  (pair-lty pax:function)
-  (pair-rty pax:function)
-  (pair-left pax:function)
-  (pair-right pax:function)
-
-  (left pax:type)
-  (left-value pax:function)
-
-  (right pax:type)
-  (right-value pax:function)
-
-  (case-on pax:type)
-  (case-on-lty pax:function)
-  (case-on-rty pax:function)
-  (case-on-cod pax:function)
-  (case-on-on pax:function)
-  (case-on-left pax:function)
-  (case-on-right pax:function)
-
-  (fst pax:type)
-  (fst-lty pax:function)
-  (fst-rty pax:function)
-  (fst-value pax:function)
-
-  (snd pax:type)
-  (snd-lty pax:function)
-  (snd-rty pax:function)
-  (snd-value pax:function)
-
-  (lamb pax:type)
-  (lamb-vty pax:function)
-  (lamb-tty pax:function)
-  (lamb-value pax:function)
-
-  (app pax:type)
-  (app-dom pax:function)
-  (app-cod pax:function)
-  (app-func pax:function)
-  (app-obj pax:function)
-
-  (index pax:type)
-  (index-index pax:function)
-
-  (typed pax:function)
-  (typed-stlc-type pax:function) (typed-stlc-value pax:function))
+  (stlc pax:class)
+  (<stlc> pax:class)
+
+  (absurd pax:class)
+  (unit pax:class)
+  (left pax:class)
+  (right pax:class)
+  (case-on pax:class)
+  (pair pax:class)
+  (fst pax:class)
+  (snd pax:class)
+  (lamb pax:class)
+  (app pax:class)
+  (index pax:class)
+
+  (absurd pax:function)
+  (unit pax:function)
+  (left pax:function)
+  (right pax:function)
+  (case-on pax:function)
+  (pair pax:function)
+  (fst pax:function)
+  (snd pax:function)
+  (lamb pax:function)
+  (app pax:function)
+  (index pax:function)
+
+  "Given (term : so0) we get ((absurd term) : tcod)"
+
+  (tcod (pax:method () (absurd)))
+  (term (pax:method () (absurd)))
+  (ttype (pax:method () (absurd)))
+
+  "We have (unit : so1)"
+  (ttype (pax:method () (unit)))
+
+  "Given (rty : Type) and (term : lty) we have  ((left rty term) : lty + rty)"
+  (rty (pax:method () (left)))
+  (term (pax:method () (left)))
+  (ttype (pax:method () (left)))
+
+  "Given (lty : Type) and (term : rty) we have ((right lty term) : lty + rty)"
+  (lty (pax:method () (right)))
+  (term (pax:method () (right)))
+  (ttype (pax:method () (right)))
+
+  "Given (on : lty + rty), (ltm : cod), (rtm : cod) produced in the context of lty and rty respectively, we get ((case-on on ltm rtm) : cod ) "
```

Comment:
It's OK to newline strings like this, it doesn't affect the formatting. only if it's 2 lines or more IIRC, so this can be newlined